lib/mixlib/install/generator/bourne/scripts/platform_detection.sh

```diff
@@ -130,12 +130,14 @@ case $platform in
     platform_version=$major_version
     ;;
   "debian")
-    # FIXME: remove client-side yolo here
-    case $major_version in
-      "5") platform_version="6";;  # FIXME: need to port this "reverse-yolo" into platform.rb
-      "6") platform_version="6";;
-      "7") platform_version="6";;
-    esac
+    if test "x$major_version" = "x5"; then
+      # This is here for potential back-compat.
+      # We do not have 5 in versions we publish for anymore but we
+      # might have it for earlier versions.
+      platform_version="6"
```

Comment:
This is tricky. Ideally this remapping as well as `platforms.rb` of omnitruck should go into `mixlib-install` as Ruby code. But we need to support `install.sh` on platforms present in [here](https://github.com/chef/omnitruck/blob/master/platforms.rb) so I'm not sure if we can get rid of this mapping altogether. 

There is definitely room for more cleanup here but I'd like to cautious here since `install.sh` is a widely used thing.